lib/golomb/golomb.js

```diff
@@ -241,26 +241,30 @@ class Golomb {
   /**
    * Instantiate a block filter from a P, 128-bit key and items.
    * @param {Number} P
+   * @param {Number} M_
    * @param {Buffer} 128-bit key
    * @param {Buffer[]} items
    * @returns {Golomb}
    */
 
-  fromItems(P, key, items) {
+  fromItems(P, M, key, items) {
```

Comment:
Is there a reason why this function only fixed M and left P variable? What I mean is, will potential future GCS filters actually change M? Or is there some reason @tuxcanfly assumed they wouldn't?